catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/report/AttributeValidationReportImpl.java

```diff
@@ -51,6 +64,20 @@ public void addSuggestedValue(final String value) {
         suggestedValues.add(value);
     }
 
+    /**
+     * Adds a a set of attribute values to the report.
+     *
+     * @param values a set of suggested attribute values to add to the report
+     * @throws IllegalArgumentException if {@code value} is null
+     */
+    public void addSuggestedValues(Set<String> values) {
+        Preconditions.checkArgument(values != null, "The suggested values cannot be null.");
+        Preconditions.checkArgument(values.size() > 0, "The suggested values cannot be null.");
```

Comment:
Cannot be empty. Should also add the non-empty requirement to the javadoc for this and `addViolations`.